uuid_maker/alignm-data_uuid.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "name": "Lawful Good",
+    "index": "lawful-good",
+    "desc": "A lawful good character typically acts with compassion and always with honor and a sense of duty. However, lawful good characters will often regret taking any action they fear would violate their code, even if they recognize such action as being good. Such characters include gold dragons, righteous knights, paladins, and most dwarves.",
+    "id": "520391c3-9cd0-4a85-98aa-81fd376218b5"
```

Comment:
why are these files doubled? Once with `id` and once without? I think you can get rid of the non-id version?